packages/hardhat-core/.eslintignore

```diff
@@ -0,0 +1 @@
+test/fixture-projects/**/*
```

Comment:
I think we can use the same as in `.prettierignore` instead:

```suggestion
/test/fixture-projects/**/artifacts
/test/fixture-projects/**/cache
```

Reply:
Fixed in 106b9c443b454483fe8bdac40f2263f85af8bc60

Reply:
I might need to revert this. If we do that, we get this error:

```
Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: test/fixture-projects/broken-typescript-config-project/hardhat.config.ts.
The file must be included in at least one of the projects provided
```

If I then use an override to disable `parserOptions.project` for the `**/fixture-projects/**/*.ts` files, eslint complains because:

```
Error while loading rule '@typescript-eslint/await-thenable': You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.
```

There surely are ways to fix this. For example, we could split the main config into two: one with the rules that require type information and the other with the rest, and then we'd only use the latter for linting the fixture projects. But I don't think the value added would be worth it.

Reply:
Tentatively reverted, but let me know if you'd rather lint those files anyway.

Reply:
tbh, now I wonder if we were ever linting those ts files.

Reverting it looks like the best trade-off.